### PR TITLE
Make noarch

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Makes this package noarch.

`pip check` will fail due to the incorrectly specified requirements of transformers 3.5.1 in conda-forge wich is why I removed it again (a8a43d5).

Most parts are generated by Grayskull.

Edit: 
Closes #7
Closes #4

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
@ndmaxar @oblute @rluria14 Just gently bumping this again, thanks!

**Comment by BastianZim** [comment]
Pinging this again. Also, let me know if you'd like some help with maintaining this package.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,52 +1,62 @@
...
-    - tqdm >=4.26.0
-    - segtok
-    - matplotlib-base >=2.2.3
-    - mpld3
-    - scikit-learn >=0.21.3
-    - sqlitedict >=1.6.0
+    - bpemb >=0.3.2
     - deprecated >=1.2.4
+    - ftfy
+    - gdown ==3.12.2
```
Comment (isuruf) [inline recipe/meta.yaml]: Why this strict?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,52 +1,62 @@
...
+    - janome
+    - konoha <5.0.0,>=4.0.0
     - langdetect
-    - sentencepiece
-    - ftfy
     - lxml
-    - gdown
-    - dataclasses  # [py36]
+    - matplotlib-base >=2.2.3
+    - mpld3 ==0.3
```
Comment (isuruf) [inline recipe/meta.yaml]: Same here

File: recipe/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipe/meta.yaml]: Comes from upstream: https://github.com/flairNLP/flair/blob/2fde64610244c7706cef68c882b9ce0e96261d2d/requirements.txt#L22

File: recipe/meta.yaml (hunk 2 above)
Comment (BastianZim) [reply recipe/meta.yaml]: Also from upstream: https://github.com/flairNLP/flair/blob/2fde64610244c7706cef68c882b9ce0e96261d2d/requirements.txt#L7

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,52 +1,62 @@
...
+    - numpy <1.20.0
```
Comment (isuruf) [inline recipe/meta.yaml]: This should be removed. The reason given at https://github.com/flairNLP/flair/commit/2355d0236f0fee813b7806f527ed52c73c4c77ae doesn't make sense to me.

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/meta.yaml]: Let's change this to `>=3.12.2`. I don't think there's a reason to do this.

File: recipe/meta.yaml (hunk 3 above)
Comment (BastianZim) [reply recipe/meta.yaml]: To a different bound or none at all?

File: recipe/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipe/meta.yaml]: Changed

File: recipe/meta.yaml (hunk 3 above)
Comment (isuruf) [reply recipe/meta.yaml]: none at all

File: recipe/meta.yaml (hunk 2 above)
Comment (isuruf) [reply recipe/meta.yaml]: let's do `>=` for all `==`

Comment (BastianZim) [reply recipe/meta.yaml]: Done

File: recipe/meta.yaml (hunk 3 above)
Comment (BastianZim) [reply recipe/meta.yaml]: Done

**Comment by BastianZim** [comment]
Changes are applied and tests are passing. Thanks for the review @isuruf

**Pull request merged**